### ISSUE TITLE
White screen

### ISSUE THREAD

**Issue by Kevgilmore** [obetomuniz/ionic-google-maps-starter#13]
When following the steps and deploying to an android device using android run, I simply get a blank white screen - no map.

Thanks

**Comment by jdkolassa** [comment]
I also noticed that **wf9a5m75/phonegap-googlemaps-plugin** doesn't exist and redirects to **mapsplugin/cordova-plugin-googlemaps**; I'm assuming that's not the issue but my first thought was if this was pointing to the right code. I did notice in package.json that "cordovaPlugins" had "locator:" pointing to the older repo, though I didn't fix it by pointing to the new one. But more than likely I'm just missing an Ionic-CLI command, so I'll go looking for that. Just a thought.
